frontend/packages/console-demo-plugin/src/dashboards/foo-health.ts

```diff
@@ -0,0 +1,12 @@
+import { HealthState } from '@console/internal/components/dashboard/health-card/states';
```

Comment:
Nit-pick: I'd just name this file `health.ts` instead of `foo-health.ts`.

Reply:
renamed to `health.ts`